tools/insmod.c

```diff
@@ -161,7 +161,7 @@ static int do_insmod(int argc, char *argv[])
 	free(opts);
 
 	log_close();
-	return err >= 0 ? EXIT_SUCCESS : EXIT_FAILURE;
+	return r == 0 ? EXIT_SUCCESS : EXIT_FAILURE;
```

Comment:
why do we have both err and r as control here? Just get rid of `err` and use `r` for the returns above?